src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -655,6 +655,11 @@ public struct StartConfiguration
             /// If true, Analytics and Usage options are hidden from UI 
             /// </summary>
             public bool HideReportOptions { get; set; }
+
+            /// <summary>
+            /// If true, autocomplete method options are hidden from UI 
+            /// </summary>
+            public bool HideAutocompleteMethodOptions { get; set; }
```

Comment:
@pinzart @mjkkirschner Is this allowed to add new params in this class?

Reply:
https://learn.microsoft.com/en-us/dotnet/core/compatibility/library-change-rules
looks at `DISALLOWED: Adding an instance field to a struct that has no nonpublic fields`
```
Adding any new fields - public or nonpublic - to such a struct is a source breaking change for these callers, as the compiler will now require the additional fields to be initialized.
```